BedrockServer.cpp

```diff
@@ -1306,6 +1306,15 @@ void BedrockServer::postPoll(fd_map& fdm, uint64_t& nextActivity) {
 
                     // Create a command.
                     BedrockCommand command(request);
+
+                    // Get the source ip of the command.
+                    unsigned long ip = ntohl(s->addr.sin_addr.s_addr);
+                    // This number is the unsigned long representation of 127.0.0.1.
+                    if (2130706433 != ip) {
+                        // If the command came from anywhere else than localhost, tag it with a source of 'www'.
+                        command.request["_source"] = "www";
```

Comment:
Lets set this to the ip of the source, and let plugins decide what that means. This is currently oddly specific to us.